apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -286,6 +286,20 @@ impl Plugin for Telemetry {
             .expect("otel error handler lock poisoned, fatal");
 
         let mut config = init.config;
+        // This code would have enabled datadog agent sampling by default, but for now we will leave it as opt-in.
+        // If the datadog exporter is enabled then enable the agent sampler.
+        // If users are using otlp export then they will need to set this explicitly in their config.
+        //
+        // if config.exporters.tracing.datadog.enabled()
+        //     && config
+        //         .exporters
+        //         .tracing
+        //         .common
+        //         .preview_datadog_agent_sampling
+        //         .is_none()
+        // {
+        //     config.exporters.tracing.common.preview_datadog_agent_sampling = Some(true);
+        // }
```

Comment:
Should we keep this ?

Reply:
Removed: [1776e59](https://github.com/apollographql/router/pull/6112/commits/1776e5983a93e523cbd9dce31dafaeefe209fb02)